lte/protos/apn.proto

```diff
@@ -65,16 +65,17 @@ message APNConfiguration {
   // Optional static IP to allocate for this subscriber on this APN
   string assigned_static_ip = 6;
 
-  message APNResource {
```

Comment:
Is this change necessary?

If yes, please explain

If no: while I agree with this stylistic change, let's defer it to a later PR since (a) we want this "backport" PR to be as minimal as possible and (b) we're refactoring apn/apn_resource soon anyway, not worth making the change at this point since Karthik will handle during the larger refactor

Reply:
I made this change because to create the `ApnResourceInternal` proto inside of `lte/cloud/go/services/subscriberdb/protos`, it needs to import the `ApnResource` proto from the `magma.lte` package, and I didn't find a good way to import a nested message (`ApnConfiguration_ApnResource`) from another package. Any tips on this to avoid this change would be awesome! @hcgatewood